input/resources/au-relatedperson.xml

```diff
@@ -18,6 +18,52 @@
+        <key value="inv-relper-0" />
+        <severity value="warning" />
+        <human value="Individual gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply" />
+        <expression value="extension('http://hl7.org/fhir/StructureDefinition/individual-genderIdentity').empty() or extension('http://hl7.org/fhir/StructureDefinition/individual-genderIdentity').value.memberOf('https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1')"/>
+        <source value="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson" />
+      </constraint>
+      <constraint>
+        <key value="inv-relper-1" />
+        <severity value="warning" />
+        <human value="Individual pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply" />
+        <expression value="extension('http://hl7.org/fhir/StructureDefinition/individual-pronouns').empty() or extension('http://hl7.org/fhir/StructureDefinition/individual-pronouns').value.memberOf('https://healthterminologies.gov.au/fhir/ValueSet/australian-pronouns-1')"/>
+        <source value="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson" />
+        </constraint>
+    </element>
+    <element id="RelatedPerson.extension">
+      <path value="RelatedPerson.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="RelatedPerson.extension:genderIdentity">
+      <path value="RelatedPerson.extension" />
+      <sliceName value="genderIdentity" />
+      <short value="The related person's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
+      <comment value="Individual Gender identity [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). See [Australian Bureau of Statics (ABS) Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) for further materials."/>
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
+      </type>
+      <condition value="inv-relper-0"/>
+    </element> 
+    <element id="RelatedPerson.extension:individualPronouns">
+      <path value="RelatedPerson.extension"/>
+      <sliceName value="individualPronouns"/>
+      <short value="The related person's pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply"/>
```

Comment:
Shorten the text:

1. Replace "The related person's pronouns" with "Pronouns".
2. Replace "any of the" with "any codes"

Reply:
Done